src/main/preload/modalPreload.js

```diff
@@ -4,7 +4,11 @@
 
 'use strict';
 
-import {ipcRenderer} from 'electron';
+import {ipcRenderer, contextBridge} from 'electron';
+
+contextBridge.exposeInMainWorld('ipcRenderer', {
```

Comment:
We don't need to expose this in the main world, see the code below. We can have specific messages sent by the front end that will do specific functions. This way we don't expose the `ipcRenderer` module to the `window` where it could be accessed via the Developer Console for example, making the approach more secure.